### PR TITLE
[Homework #20] - Routing-redux by Abraham Trenyan

### PR DESCRIPTION
**Pull request by AbrahamTrenyan**

### CODE REVIEW

File: homeworks/abraham-trenyan/20-routing-redux/todo-app/src/components/TodoList/TodoList.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Task from '../Task/Task';
+import { useSelector } from 'react-redux';
+
+const TodoList = () => {
+ const todos = useSelector((state) => {
```
Comment (VitalikMelnyk) [inline homeworks/abraham-trenyan/20-routing-redux/todo-app/src/components/TodoList/TodoList.jsx:6]: To be honest, if I don't write additional code inside function like console.log or extra manipulation
I use the short arrow function without return and {}.
Your code will be cleaner and have fewer lines of code.
`const todos = useSelector((state) => state.tasks)`

Comment (VitalikMelnyk) [reply homeworks/abraham-trenyan/20-routing-redux/todo-app/src/components/TodoList/TodoList.jsx:6]: Please, make it suggestion in your functions.

**Comment by VitalikMelnyk** [comment]
Hello @AbrahamTrenyan 
The functionality looks good. Please check my notes and make changes.
Thanks

**Comment by AbrahamTrenyan** [comment]
Ive refactored the arrow functions to the shorter syntax, thanks for the suggestion.

**Review by VitalikMelnyk — APPROVED** [review]

**Pull request merged**